web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -197,32 +203,37 @@
               }
             }
 
-            // layers
+            // load layers from Settings
             if (config.layers && config.layers.length) {
               config.layers.forEach(function(layerInfo) {
                 gnMap.createLayerFromProperties(layerInfo, map)
                   .then(function(layer) {
                     if (layer) {
-                      map.addLayer(layer);
+                      layer.displayInLayerManager = false;
```

Comment:
Layers from settings should be background layers as they are loaded by default. Before this, they were shown on the map but not shown either on the background list of layers or the layer manager, which was weird.